document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresClient.java

```diff
@@ -51,6 +51,23 @@ public Connection getPooledConnection() throws SQLException {
     return connectionPool.getConnection();
   }
 
+  public void close() {
+    if (connection != null) {
+      try {
+        log.info("Closing connection #{} to {}", count, connectionConfig.toConnectionString());
```

Comment:
What the `count` variable here? number of connection in the pool?

Reply:
The existing method moved here since it was made public. But the count refers to the # of connections already made.